technology-adapter-api.md

```diff
@@ -259,3 +259,17 @@ This will produce a report similar to this:
   "tags": []
 }
 ```
+
+### Identify
+
+Allows the user to identify the hub. Following sequence will be repeated three times - a light will turn on for 600ms then turn off for 200ms. 
```

Comment:
The hub or the device?

Reply:
The hub, it's a requirement for Z-wave certification as far as I know